src/App.tsx

```diff
@@ -43,6 +44,7 @@ function App() {
           <Route path="*" element={<Page404 />}></Route>
           <Route path="/contact" element={<Contact />}></Route>
           <Route path="/manage_users" element={<AssignRole />} />
+          <Route path="/checkout" element={<Checkout />}></Route>
         </Routes>
```

Comment:
its better you protect this route also

because on checkout page whenever  you type any data without being logged in 
numerous toasts are being displayed which is a bit not well